airbyte-integrations/bases/base-normalization/normalization/transform_catalog/stream_processor.py

```diff
@@ -339,21 +359,27 @@ def generate_json_parsing_model(self, from_table: str, column_names: Dict[str, T
         )
         return sql
 
+    def get_emitted_at(self, in_jinja: bool = False):
+        return self.name_transformer.normalize_column_name(self.airbyte_emitted_at, in_jinja, False)
+
     def extract_json_columns(self, column_names: Dict[str, Tuple[str, str]]) -> List[str]:
         return [
-            StreamProcessor.extract_json_column(field, self.json_column_name, self.properties[field], column_names[field][0], "table_alias")
+            self.extract_json_column(field, self.json_column_name, self.properties[field], column_names[field][0], "table_alias")
             for field in column_names
         ]
 
-    @staticmethod
-    def extract_json_column(property_name: str, json_column_name: str, definition: Dict, column_name: str, table_alias: str) -> str:
+    def extract_json_column(self, property_name: str, json_column_name: str, definition: Dict, column_name: str, table_alias: str) -> str:
         json_path = [property_name]
         # In some cases, some destination aren't able to parse the JSON blob using the original property name
         # we make their life easier by using a pre-populated and sanitized column name instead...
         normalized_json_path = [transform_json_naming(property_name)]
         table_alias = f"{table_alias}"
         if "unnested_column_value" in json_column_name:
             table_alias = ""
+        if json_column_name.startswith("'_") and self.destination_type == DestinationType.ORACLE:
+            json_column_name = json_column_name.replace("'", "")
+            json_column_name = self.name_transformer.normalize_column_name(json_column_name, True, False)
+
```

Comment:
this looks weird, why do you need to remove all quotes from the column name?

Reply:
the `_json_data` is retrived as `'_json_data'` so I need to apply this operation and use quote with Oracle.

Reply:
I the function is apply directly in the variable will transform to `__json_data_`... because the chars not allowed in Oracle.

Reply:
I removed this code and make the logic when creating the StreamProcessor.